### PR TITLE
Allow deploying to Heroku

### PR DESCRIPTION
**Pull request by chrisroos**
The current mechanism for making changes to Smart Answers isn't ideal: We have to duplicate everything about the Smart Answer, make the changes and then deploy to preview for someone to view and test. See "[Making bigger changes](https://github.com/alphagov/smart-answers#making-bigger-changes)" for more information.

We think that using git branches, instead of manually branching in the filesystem, will make our lives easier in this situation. Although we can deploy branches to Preview, the auto-deployment functionality means that we can't guarantee how long they'll live. Allowing us to deploy to Heroku should solve this particular problem.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,3 +95,17 @@ To help developers track changes in files easily, it is best if you commit V2 fi
 ## Detailed documentation
 
 - [How to archive a Smart Answer](doc/archiving.md)
+
+## Deploying to Heroku
+
+    heroku apps:create
+
+    heroku config:set GOVUK_APP_DOMAIN=preview.alphagov.co.uk
```
Comment (tadast) [inline README.md]: Multiple variables can be set in one command, would make things faster.

File: README.md [hunk 2]
```diff
@@ -95,3 +95,17 @@ To help developers track changes in files easily, it is best if you commit V2 fi
 ## Detailed documentation
 
 - [How to archive a Smart Answer](doc/archiving.md)
+
+## Deploying to Heroku
+
+    heroku apps:create
```
Comment (tadast) [inline README.md]: Heroku(Amazon) has servers in Europe, so `heroku apps:create --region eu` would have lower latency

**Comment by bradleywright** [comment]
I am very much :+1: to this workflow improvement.

**Comment by tadast** [comment]
This is looking great <3 Can't wait to try this out

**Comment by bradleywright** [comment]
Worth making a shell script called `./heroku-startup.sh` or similar that contains the env variables? And then update the instructions to use that instead.

File: README.md (hunk 2 above)
Comment (chrisroos) [inline README.md]: Cool. I didn't know that. Thanks, @tadast.

**Comment by chrisroos** [comment]
> Worth making a shell script called ./heroku-startup.sh or similar that contains the env variables? And then update the instructions to use that instead.

@bradleywright: Absolutely. We figured we'd go with the README initially but then wrap it all in a script if we go ahead with this change.

**Comment by chrisroos** [comment]
I'm going to merge these changes to master.

**Comment by chrisroos** [comment]
Manually merged in 849ff4057ae2db762d93685f2a06396f2675da19.

**Pull request closed without merging**